etc/ubsan-suppr.txt

```diff
@@ -0,0 +1,5 @@
+# Undefined behavior sanitizer suppressions
+# See here for how this works:
+#   https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html
+
+pointer-overflow:be_visitor_operation_upcall_command_ss::gen_upcall
```

Comment:
Does this mean that `tao_idl` has undefined behavior? Do we need a new Issue in that repo?

Reply:
Technically yes:

https://github.com/DOCGroup/ACE_TAO/blob/466542bd7d6d5b23783163fe420c2ea986734177/TAO/TAO_IDL/be/be_visitor_operation/upcall_command_ss.cpp#L223-L225

If `strstr` returns null, then it's doing pointer arithmetic with null, which for some reason is under the ubsan category called `pointer-overflow`. I didn't think it was worth fixing, at least not yet as it seems harmless because it's only doing a comparison. Also there's probably a lot more stuff to fix at runtime so I think we can defer fixing this until we decide to properly tackle undefined behavior.